generator/generator.yml

```diff
@@ -132,11 +132,34 @@ modules:
 # Tested on Arista DCS-7010T-48 switch
 #
   arista_sw:
-    walk: 
+    walk:
       - sysUpTime
       - interfaces
       - ifXTable
       - 1.3.6.1.2.1.25.3.3.1.2 # hrProcessorLoad
       - 1.3.6.1.2.1.25.2.3.1.6 # hrStorageUsed
       - 1.3.6.1.4.1.30065.3.1.1 # aristaSwFwdIp
 
+# DD-WRT
+#
+# The list of SNMP OIDs to care about for DD-WRT can be found here: https://www.dd-wrt.com/wiki/index.php/SNMP#Known_OID.C2.B4s_via_SNMP
+#
+# Tested on  DD-WRT v3.0-r31825 (04/06/17) with an ASUS RT-AC66U
+
+  ddwrt:
+    walk:
+      - sysUpTime
+      - interfaces
+      - ifXTable
+      - 1.3.6.1.2.1.25.2 # hrStorage
+      - 1.3.6.1.4.1.2021.4 # ucdavis mem
+      # ucdavis laTable (load avg)
+      - 1.3.6.1.4.1.2021.10.1.1 # laIndex
+      - 1.3.6.1.4.1.2021.10.1.2 # laNames
+      - 1.3.6.1.4.1.2021.10.1.5 # laLoadInt
+      - 1.3.6.1.4.1.2021.11 # ucdavis systemStats
```

Comment:
The convention in this file is for the comment to be just the object name, without any reference to the module name.